config/locales/en.yml

```diff
@@ -323,6 +323,53 @@ en:
       title: 'Title'
       teacher: 'Teacher'
       editors: 'Editors'
+    annotation:
+      annotations: 'Annotations'
+      annotation_modal_head: 'Create annotation'
+      inherit_from_lecture: 'inherit from lecture'
+      time: 'Time:'
+      comment: 'Comment'
+      color: 'Color'
+      category: 'Category'
+      note: 'note'
+      content: 'content'
+      mistake: 'mistake'
+      presentation: 'presentation'
+      where_problem: >
+        Where's the problem?
+      definition: 'definition'
+      argument: 'argument'
+      strategy: 'proof strategy'
+      previous_annotation: 'previous annotation'
+      annotation_position: 'go to annotation'
+      edit_annotation: 'edit annotation'
+      close_annotation_area: 'close area'
+      next_annotation: 'next annotation'
+      further_help: 'You need further help? Then click on:'
```

Comment:
If prof did not enter a link at all, you still get shown this `further_help` message, but without a link, which is not ideal. Maybe only show this message if a link has been entered by the prof?